### DISCUSSION TITLE
Installation on M1 mac

### DISCUSSION THREAD

**Discussion by lenne0815** [8ff/firescrew#16] [Q&A] [answered]
Hello, i would love to try firescrew (when i get it working connect it via mqtt to ha) but im already lost at the very first step, how to install it on a m1 mac ?

**Comment by chiefMarlin** [comment, marked as answer]
To use baremetal acceleration on mac its best to run single binary (as docker does not support gpu pass through on mac)
So here are the steps

### Install ffmpeg
apt-get install ffmpeg
### Download binary
curl -O firescrew -L https://github.com/8ff/firescrew/releases/download/latest/firescrew.linux.amd64 && chmod +x firescrew

### Generate default config
./firescrew -t > config.json

### Edit config.json with your own values

### Start object detection
./firescrew config.json

### Start WebUI to see the events
./firescrew -s rec/hi :8080
